openedx/core/djangoapps/programs/utils.py

```diff
@@ -62,11 +65,26 @@ def get_program_and_course_data(site, user, program_uuid, mobile_only=False):
     return program_data, course_data
 
 
+def get_buy_subscription_url(*skus, program_uuid=None):
+    """
+    Construct the url for the subscription purchase page.
+    """
+    query_params = {'sku': skus}
+    print(f"Query Params are: {query_params}")
+    url = '{buy_subscription_path}/{program_uuid}/?{query_params}'.format(
+        buy_subscription_path=settings.SUBSCRIPTIONS_BUY_SUBSCRIPTION_URL,
+        program_uuid=program_uuid,
+        query_params=urlencode(query_params, doseq=True),
+    )
```

Comment:
Nit: This is the old way to format strings. Should we be using f-strings here?

Reply:
The API endpoint specifications changes a bit and we have completely removed this utility method now.